specutils/tests/spectral_examples.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+import astropy.units as u
+from astropy.modeling import models
+from specutils.spectra import Spectrum1D
+import pytest
+
+class SpectraExamples(object):
```

Comment:
The summary of our discussion offline. This is a fixture container class that has several spectra defined in it.  This makes it easier to pass in the wavelengths, flux and resulting Spectrum1D objects for easier testing.  I added documentation regarding this since our discussion.